Dockerfile

```diff
@@ -21,11 +21,13 @@ RUN apk add -q --no-cache \
   sed \
  && pip3 install -q docker-compose
 
+COPY get-package-details.sh /usr/bin/get-package-details
+
 RUN wget -q "https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl" -O "/usr/bin/kubectl" \
  && wget -q "https://github.com/UKHomeOffice/kd/releases/download/v${KD_VERSION}/kd_linux_amd64" -O "/usr/bin/kd" \
  && chmod +x "/usr/bin/kubectl" \
- && chmod +x "/usr/bin/kd"
-
-COPY get-package-details.sh /get-package-details.sh
+ && chmod +x "/usr/bin/kd" \
+ && echo '' \
+ && echo 'source /usr/bin/get-package-details' >> /root/.bashrc
```

Comment:
I wonder whether Drone uses bash or whether it uses sh.

Reply:
Well we use `bash`, so it doesn't really matter:
```Dockerfile
CMD ["bash"]

Reply:
Here: 
https://github.com/UKHomeOffice/lev-ci/blob/e3cc811b286659e3cee26f74bbe02a070f36aa9b/Dockerfile#L31

Reply:
I think Drone overwrites the entrypoint.